src/module/BTCRateProvider.sol

```diff
@@ -29,3 +34,32 @@ contract BTCRateProvider is BaseKernelRateProvider {
         revert UnsupportedAsset(asset);
     }
 }
+
+contract TestnetBTCRateProvider is BaseKernelRateProvider {
```

Comment:
Let's move these in a test/module/ directory

so we keep the /src directory clean of any code that doesn't need an audit.

Mixing them together will confuse our auditors as the testnet code doesn't need one

Reply:
fixed in [03af3e0](https://github.com/yieldnest/yieldnest-kernel-lrt/pull/20/commits/03af3e0fa5af38c8eac25950303b05f0e2b9abe1)